.gitignore

```diff
@@ -3,3 +3,4 @@ output.xml
 report.html
 env.list
 resin.img
+/tmp
```

Comment:
This doesnt seem to be needed unless we in / - In which case we have a lot more things to cleanup.